pkg/config/config.go

```diff
@@ -43,6 +43,13 @@ type Config struct {
 	Nodename string
 }
 
+const (
+	Quota    = "quota"
+	RefQuota = "refquota"
+)
```

Comment:
Please add comments explaining them.

Reply:
i have added comments for quota types @Abhinandan-Purkait